package.json

```diff
@@ -84,6 +84,7 @@
     "eslint-plugin-ava": "5.1.0",
     "eslint-plugin-babel": "5.1.0",
     "eslint-plugin-chai-expect": "1.1.1",
+    "eslint-plugin-eslint-comments": "^3.0.1",
```

Comment:
The other dependencies are pinned to a specific version, maybe this one should be as well?